### DISCUSSION TITLE
Sign in failed: Error: API rate limit exceeded for user ID

### DISCUSSION THREAD

**Discussion by lexoyo** [community/community#75007] [Copilot Conversations]
### Select Topic Area

Question

### Body

Sign in failed: Error: API rate limit exceeded for user ID 715377. If you reach out to GitHub Support for help, please include the request ID C638:EBCF:1EA05F7F:1EFBF3F1:654DF726.

**Comment by lexoyo** [comment]
I had to register to so called copilot labs, i don't know why and i wish i didn't

**Reply by emicovi** [reply]
I have the same error.. how did you fix it?

**Reply by lexoyo** [reply]
Sorry i don't remember even posting this

**Comment by lionello** [comment]
In my case there was a 3rd party site that I connected to GitHub and it was excessively spamming the GitHub REST API on my behalf. This caused my account to be throttled. Check your integrations here: https://github.com/settings/installations
